src/providers/providers/index.ts

```diff
@@ -146,26 +146,12 @@ export const BINANCECHAINWALLET: IProviderInfo = {
   check: "isBinanceChainWallet"
 };
 
-/**
- * @deprecated Use CoinbaseWalletSdk
- */
-export const WALLETLINK: IProviderInfo = {
```

Comment:
Note: this is required because it iterates through the `id` from the object. Otherwise, this will silently break for users.